buildpack-deps/README.md

```diff
@@ -53,7 +53,7 @@ You are invited to contribute new features, fixes, or updates, large or small;
 we are always thrilled to receive pull requests, and do our best to process them
 as fast as we can.
 
-Before you start to code, we recommend discussing your plans 
+Before you start to code, we recommend discussing your plans
```

Comment:
You're talking about this one specific space being generated as part of the template script (due to the whole mailing-list insertion logic)?